recipe/meta.yaml

```diff
@@ -10,15 +10,21 @@ source:
   sha256: 34ff698c870c87e6e47a838eeaaae729fa73349139fc8db12211d2a22b78af6b
 
 build:
-  number: 0
+  number: 1
   skip: True  # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - autoconf
+    - automake
+    - gettext-tools
```

Comment:
If you add that here, you probably also need `libintl-devel  # [osx]` in host

Reply:
For this particular case, `gettext-tools` is needed for `autopoint` used by `autoreconf`.  The only shared library linked to by `wdiff` on macOS is `ncurses` - it doesn't link to `libintl`.  So I think this should only be in build for this recipe?

Reply:
From the `osx-64` logs:

> checking for GNU gettext in libintl... no

You need `libintl-devel` on osx to get translations. This isn't required on linux as the functionality is part of glibc.

Reply:
I see, thank you for catching that!  I’ll update accordingly.  That’s also useful for some other recipes I’ll be submitting soon that have the same dependency.

Reply:
Please see the last commit for the fix - `libintl` is now detected on macOS and is dynamically linked during the build.